app/models/transport.rb

```diff
@@ -1,7 +1,9 @@
 class Transport < ApplicationRecord
-  TRANSPORT_NAMES = %i[car train bus no_transport unknown].freeze
-
-  enum name: TRANSPORT_NAMES
-
-  validates_presence_of :name
+  enum name: {
+    car: 'Can',
```

Comment:
May be ```Car``` need here?

Reply:
Sure...

Reply:
https://github.com/vova/war_shelter/pull/26/commits/4fba1553e3a87f5a1322fa8ee4330cbd9641736f

Reply:
https://github.com/vova/war_shelter/pull/26/commits/4fba1553e3a87f5a1322fa8ee4330cbd9641736f